.sun-ci.yml

```
@@ -0,0 +1,90 @@
+default:
+  workspace: shared
+
+stages:
+  - build
+  - check
+
+jobs:
+  - name: Build Laravel
+    stage: build
+    image: ghcr.io/avengerscodelovers/php:8.2-ci
+    services:
+      - image: mysql:8.0
+        name: mysql
+        environment:
+          MYSQL_DATABASE: app_db
+          MYSQL_USER: app_user
+          MYSQL_PASSWORD: secret
+          MYSQL_ROOT_PASSWORD: root
+    environment:
+      APP_ENV: testing
+      COMPOSER_MEMORY_LIMIT: 2G
+      COMPOSER_ALLOW_SUPERUSER: "1"
+    cache:
+      - key: composer_vendor_$CI_BRANCH
+        paths:
+          - vendor
+    script:
+      - cp .env.testing.example .env
+      - composer install
+      - php artisan key:generate
+      - php artisan migrate
+      - php artisan route:cache
```

chơi hẳn lệnh [optimize](https://github.com/laravel/framework/blob/11.x/src/Illuminate/Foundation/Console/OptimizeCommand.php#L17) đi cho máu :v 
```suggestion
      - php artisan optimize
```